src/app/vos/pages/group-detail-page/group-members/group-members.component.ts

```diff
@@ -33,8 +35,17 @@ export class GroupMembersComponent implements OnInit {
   firstSearchDone = false;
 
   loading = false;
+  data: string;
```

Comment:
```suggestion
  data: 'search' | 'all';
```
You can define the exact values that are only allowed in this variable.